csi.proto

```diff
@@ -317,6 +348,21 @@ message VolumeCapability {
 
   // This is a REQUIRED field.
   AccessMode access_mode = 3;
+
+
+  // Specifies how a volume can be expanded
+  message Expandable {
+    enum OperationMode {
+      UNKNOWN = 0;
+
+      // Can be expanded when in-use by a workload.
+      ONLINE = 1;
+
+      // Can only be expanded when not in-use
+      OFFLINE = 2;
+    }
+  }
+  Expandable expandable = 4;
```

Comment:
Add comment indicating when it is required.

Reply:
Fixed.